xtuner/dataset/internlm.py

```diff
@@ -48,15 +48,15 @@ def process(dataset_folder=None,
     dataset = dataset.map(
         partial(add_labels, max_length=max_length),
         remove_columns=list(dataset.column_names),
-        num_proc=num_proc)
+        map_num_proc=map_num_proc)
```

Comment:
`dataset.map(num_proc=map_num_proc)` or `dataset.map(map_num_proc=map_num_proc)`?

Reply:
It should be `dataset.map(num_proc=map_num_proc)`. Fixed!